pkgs/apps/pysisyphus/default.nix

```
@@ -0,0 +1,158 @@
+{ fetchPypi, fetchFromGitHub, buildPythonPackage, lib, writeTextFile, writeScript, makeWrapper,
+ # Python dependencies
+ autograd, dask, distributed, h5py, jinja2, matplotlib, numpy, natsort, pytest, pyyaml, rmsd, scipy,
+ sympy, scikitlearn, qcengine, ase, xtb-python, openbabel-bindings,
+ # Runtime dependencies
+ bash, jmol, multiwfn, xtb, openmolcas, pyscf, psi4, wfoverlap, nwchem, orca ? null ,
+ turbomole ? null, gaussian ? null, gamess-us ? null, cfour ? null, molpro ? null,
+ # Test dependencies
+ openssh,
+ # Configuration
+ fullTest ? false
+}:
+let
+  psi4Wrapper = writeScript "psi4.sh" ''
+    #!${bash}/bin/bash
+    ${psi4}/bin/psi4 -o stdout $1
+  '';
+  pysisrc =
+    let
+      gaussian16Conf = {
+        cmd = "${gaussian}/bin/g16";
+        formchk_cmd = "${gaussian}/bin/formchk";
+        unfchk_cmd = "${gaussian}/bin/unfchk";
+      };
+      openmolcasConf = {
+        cmd = "${openmolcas}/bin/pymolcas";
+      };
+      orcaConf = {
+        cmd = "${orca}/bin/orca";
+      };
+      psi4Conf = {
+        cmd = "${psi4Wrapper}";
+      };
+      xtbConf = {
+        cmd = "${xtb}/bin/xtb";
+      };
+      wfoverlapConf = {
+        cmd = "${wfoverlap}/bin/wfoverlap.x";
+      };
+      multiwfnConf = {
+        cmd = "${multiwfn}/bin/Multiwfn";
+      };
+      jmolConf = {
+        cmd = "${jmol}/bin/jmol";
+      };
+      text = with lib.lists; lib.generators.toINI {} (builtins.listToAttrs ([
+        { name = "openmolcas"; value = openmolcasConf; }
```

```suggestion
        { name = "openmolcas"; value.cmd = "${openmolcas}/bin/pymolcas"; }
```
This pattern would save a lot of space here.